### PR TITLE
Various upgrades

### PR DESCRIPTION
**Pull request by kwatters**
this upgrades versions of javacv , tesseract , deeplearning4j , solr and junit.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -65,1442 +65,1442 @@
...
+  </dependency>
+  <dependency>
+    <groupId>org.jmonkeyengine</groupId>
+    <artifactId>jme3-desktop</artifactId>
+    <version>3.3.2-stable</version>
+    <scope>provided</scope>
+  </dependency>
+  <dependency>
+    <groupId>org.jmonkeyengine</groupId>
+    <artifactId>jme3-lwjgl3</artifactId>
```
Comment (supertick) [inline pom.xml]: Your putting lwjgl3 back in ?  It will break oculus .. again...

File: pom.xml [hunk 2]
```diff
@@ -65,1442 +65,1442 @@
...
+  <dependency>
+    <groupId>net.sf.opencsv</groupId>
+    <artifactId>opencsv</artifactId>
+    <version>2.3</version>
+    <scope>provided</scope>
+  </dependency>
+<!-- CsvConnector end -->
+
+<!-- Deeplearning4j begin -->
+  <dependency>
```
Comment (supertick) [inline pom.xml:244]: Do you know why the whitespace difference, was this generated

**Review by supertick — CHANGES_REQUESTED** [review]
we should sync on deps for lwjgl

File: pom.xml (hunk 1 above)
Comment (kwatters) [reply pom.xml]: merge problem.. will fix. good catch.

File: pom.xml (hunk 2 above)
Comment (kwatters) [reply pom.xml:244]: Was the file generated 100% before?  This file is the result of running MavenWrapper.xml  no changes.

**Pull request merged**